### DISCUSSION TITLE
v3.1.1

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#124] [Announcements]
### New features

- Support setting opacity for bullet comments 
- Support editing cookie manually 
- Support 1080p for anonymous watching (BiliBili)

### Bug fixes

- Support auto enqueue next partition and avoid use partitions as related items
- Only 1 page of search results are shown(BiliBili)
- HTML strings in search results are not decoded (BiliBili)

Which APK should you download?

- arm64-v8a should work on most devices.
- armeabi-v7a if your device is a 10+ year old phone.
- x86_64 for 64-bit Android simulators (on a PC).
- x86 for 32-bit simulators.

If these rules not working, download the universal APK. It should work on any device.
All APKs provide the same contents and functions, but the universal one may be slower than the split ones.

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/PipePipe/releases/tag/v3.1.1'>v3.1.1</a>.</em>

**Comment by RE1AX1** [comment]
Desires for the next version.
YouTube:
1. Add full-fledged data transfer from new versions of NewPipe
2. Optimize resources for fast loading, since the application is a bit slow, launching video is a bit slower than in NewPipe. Thereby reducing the weight of the application.
3. Remove the initial splash screen when launching the application
4. Add a sponsor block
5. Add font size customization for comments.
6. Add the ability to view dislikes

**Reply by InfinityLoop1308** [reply]
1, 4, 5 -> existed issues
2 -> That is because the different design of comments
3, 6 -> Possible if I can found easy fixes, especially if there is already a pr in NewPipe's repo

**Reply by Madgitu** [reply]
Bro iam very thankful to you for this initiative.Iam in love with this app.
Can you please add a sponsorblock to this app?? 🥺

**Comment by Madgitu** [comment]
Hi bro,Firstly I thank you for this app❤️. I have a suggestion. Can you Please add a Sponsorblock to this app😢. It will be a great help🥺
